backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -702,4 +705,51 @@ private UserInfo consolidateUser(
 
     return new UserInfo(apiUser, Optional.of(orgRoles), isSiteAdmin, userStatus);
   }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public List<String> moveUserToNewOrganization(
+      String userToMoveEmail,
+      String newOrgExternalId,
+      Boolean allFacilitiesAccess,
+      OrganizationRole newRole) {
+    return moveUserToNewOrganization(
+        userToMoveEmail, newOrgExternalId, allFacilitiesAccess, Optional.empty(), newRole);
+  }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public List<String> moveUserToNewOrganization(
+      String userToMoveEmail,
+      String newOrgExternalId,
+      Boolean allFacilitiesAccess,
+      Optional<List<UUID>> optFacilityList,
+      OrganizationRole newRole) {
+
+    Boolean facilityListIsNullOrEmpty =
```

Comment:
let me know if there's a better way to express this idea. Basically we want to check whether the list is null (because Optional) or empty because in those cases, `allFacilitiesAccess` needs to be true

Reply:
this is much cleaner, thank you!